src/flint/fq_default.jl

```diff
@@ -271,12 +286,12 @@ end
 
 function _get_raw_type(::Type{fpField}, F::FqField)
   @assert _fq_default_ctx_type(F) == 4
-  return GF(UInt(order(F)))
+  return Native.GF(UInt(order(F)))
 end
 
 function _get_raw_type(::Type{FpField}, F::FqField)
   @assert _fq_default_ctx_type(F) == 5
-  return GF(order(F))
+  return Native.GF(order(F))
```

Comment:
Why do you need to prefix all of these statements by `Native.`? I thought this PR replaced the implementation of `GF` to only have one type of finite field getting constructed.
But trying `GF(5)` and `Native.GF(5)` gives me different types.

Could you explain what is happening here?

Reply:
Or is this just temporarily to get this going and in the near future all occurrences of `Native.` get removed one by one?

Reply:
We need to keep the raw/alternative/old finite fields around, mainly for internal use. The constructors are hidden from the user behind `Native.`. This internal function here explicitly asks for a field of type `FpField` (old), so it is constructed via `Native.GF`.

Does that make sense?